Makefile

```diff
@@ -65,6 +65,7 @@ guess_platform_script := \
 guess_platform := $(shell $(guess_platform_script))
 
 all: all-$(guess_platform)
+check: check-$(guess_platform)
 
```

Comment:
It might be worth adding:

````Makefile
$(MAKE) -C tests/lcb
$(MAKE) -C tests/lcs
````

as commands in the `check` rule, just so that `make check` becomes an easy way to run all the tests.